js/test/components/operatorSignIn/attestation.test.tsx

```diff
@@ -216,4 +227,42 @@ describe("Attestation", () => {
     expect(onComplete).not.toHaveBeenCalled();
     jest.useRealTimers();
   });
+
+  describe("expiry", () => {
```

Comment:
`attestation.test.tsx` now has a test for the expired case. But does it need a test that it shows the about to expire warning? I see that the details of that situation are asserted in `expiry.test.tsx`, but I'm thinking about a test that proves that `CertificationBoxes` is called.